index.html

```diff
@@ -10,32 +10,39 @@
     <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <div id="heading">
-      <header class="header-nav">
-        <a href="#" class="logo">MER<span>cy</span></a>
-        <nav id="hamburger">
-          <img src="/assets/images/bars.png" alt="menu" />
-        </nav>
-      </header>
-
-      <section class="heading">
-        <div class="header">
-          <h1>I’m MERCY <br />Glad to see <br />you!</h1>
+    <header class="landing-page">
```

Comment:
- [ ] Currently, your whole design is not responsive and it is taking the exact width of 1500px which is making it unresponsive and user needs to zoom-out to see all the contents of your website when the user is on a display less than `1500px` wider. so kindly strive to make your design responsive on all screens.
- [ ] Also at `666px` and less, the hamburger button breaks so kindly consider fixing it for the better user experience.